rskj-core/src/test/java/co/rsk/peg/StandardMultisigFederationTest.java

```diff
@@ -99,193 +110,212 @@ void membersImmutable() {
     }
 
     @Test
-    void redeemScript() {
-        final List<Integer> calls = new ArrayList<>();
-        try (MockedStatic<ScriptBuilder> scriptBuilderMocked = mockStatic(ScriptBuilder.class)) {
-            scriptBuilderMocked.when(() -> ScriptBuilder.createRedeemScript(any(int.class), any(List.class))).thenAnswer((invocationOnMock) -> {
-                calls.add(1);
-                int numberOfSignaturesRequired = invocationOnMock.<Integer>getArgument(0);
-                List<BtcECKey> publicKeys = invocationOnMock.getArgument(1);
-                Assertions.assertEquals(3, numberOfSignaturesRequired);
-                Assertions.assertEquals(5, publicKeys.size());
-                for (int i = 0; i < sortedPublicKeys.size(); i++) {
-                    Assertions.assertArrayEquals(sortedPublicKeys.get(i).getPubKey(),
-                            publicKeys.get(i).getPubKey());
-                }
-                return new Script(new byte[]{(byte) 0xaa});
-            });
-            Assertions.assertArrayEquals(federation.getRedeemScript().getProgram(), new byte[]{(byte) 0xaa});
-            // Make sure the script creation happens only once
-            Assertions.assertEquals(1, calls.size());
-        }
-    }
-
-    @Test
-    void getP2SHScript() {
-        ScriptBuilder p2shScriptBuilder = new ScriptBuilder();
-        p2shScriptBuilder.op(ScriptOpCodes.OP_HASH160);
-        p2shScriptBuilder.data(Hex.decode("57f76bf3ab818811c740929ac7a5e3ef8c7a34b9"));
-        p2shScriptBuilder.op(ScriptOpCodes.OP_EQUAL);
-
-        Script p2shScript = p2shScriptBuilder.build();
+    void testEquals_basic() {
+        assertEquals(federation, federation);
 
-        Assertions.assertEquals(federation.getP2SHScript(), p2shScript);
+        assertNotEquals(null, federation);
+        assertNotEquals(federation, new Object());
+        assertNotEquals("something else", federation);
```

Comment:
Same here. This is redundant.

Reply:
yes, i agree. but this is a test that was there before and we decided to "just let it there"